doc/dune.inc

```diff
@@ -0,0 +1,1039 @@
+(rule
```

Comment:
This one looks uselessy very heavy. Couldn't we simply use a makefile with a rule `dune exec ...src/bin/liquidsoap.exe ...` which will force the compilation of liquidsoap anyway? And it's even worse for the gneration of files with pandoc below. `make` is still useful sometimes :)

Reply:
This file is auto-generated via `dune build @gendune --auto-promote`